src/timeMachine/components/Vis.tsx

```diff
@@ -83,16 +90,41 @@ const TimeMachineVis: SFC<Props> = ({
         >
           {isViewingRawData ? (
             <AutoSizer>
-              {({width, height}) =>
-                width &&
-                height && (
+              {({width, height}) => {
+                if (isFlagEnabled('flowsUiPagination')) {
```

Comment:
maybe i'm kind of getting it. the flag has nothing to do with the feature, but instead the "is data csv or fromfluxresult" and i'd argue that the migration strategy in giraffe mandates that we allow for both, instead of overloading the data prop with some modality

so add a new prop called something else? parsedData?